Scenarios/SelfHostedGitHubRunner.md

```diff
@@ -7,7 +7,22 @@ GitHub runners can be registered for an organization (accessible for all reposit
 
 1. Navigate to [https://github.com/organizations/{organization}/settings/actions/runners/new](https://github.com/organizations/{organization}/settings/actions/runners/new) to create a self-hosted runner for your organization. Use [https://github.com/{organization}/{repository}/settings/actions/runners](https://github.com/{organization}/{repository}/settings/actions/runners) to create a self-hosted runner for a single repository.
 ![Organization new runner](https://github.com/microsoft/AL-Go/assets/10775043/f09af5ee-73b5-40e3-bad1-98f0c7b0ddaa)
-1. To create a self-hosted runner manually, choose Windows under Runner Image and x64 in architecture and follow the description on how to create a self-hosted runner manually, then go to step 9 to continue the configuration.
+1. Now, you can either [Create your self-hosted runner manually](#create-your-self-hosted-runner-manually) or you can [Use the Azure VM Template to create your self-hosted runner](#use-the-azure-vm-template-to-create-your-self-hosted-runner)
+
+## Create your self-hosted runner manually
+1. To create a self-hosted runner manually, choose Windows under Runner Image and x64 in architecture and follow the description on how to create a self-hosted runner manually
+1. Make sure that the following software is installed on the computer:
+   - Docker (I use [this script](https://github.com/microsoft/nav-arm-templates/blob/master/InstallOrUpdateDockerEngine.ps1) to install or update Docker Engine on the Azure VM)
+   - The AZ PowerShell module (I use `Install-Module az -force`)
+   - GIT (I use `choco install git --force --params "/NoAutoCrlf"` (after installing choco using `https://chocolatey.org/install.ps1`))
+   - 7zip (I use `choco install 7zip`)
+   - GitHub CLI (I use `choco install gh`)
+   - PowerShell 7 (I use `choco install pwsh -y`)
+   - Microsoft Visual C++ Redistributable for Visual Studio 2015-2022 14.36.32532 (I use `choco install vcredist140 -y`)
+
```

Comment:
.NET SDK? I don't think signing will work without it

Reply:
Which version of .net? or .net sdk, do you know?
PowerShell 7 will pull .net core along